website/docs/d/team_access.html.markdown

```diff
@@ -41,4 +41,4 @@ The `permissions` block contains:
 * `state_versions` - The permissions granted to state versions. Valid values are `none`, `read-outputs`, `read`, or `write`
 * `sentinel_mocks` - The permissions granted to Sentinel mocks. Valid values are `none` or `read`
 * `workspace_locking` - Whether permission is granted to manually lock the workspace or not.
-* `run_tasks` - Whether permission is granted to manage workspace run tasks or not.
+* `run_tasks` - (Required) Boolean determining whether or not to grant the team permission to manage workspace run tasks.
```

Comment:
Given this is a computed attribute and not an argument of the `team_access` data source, we don't need to specify `(Required)`.

Reply:
Ah, I was wondering what the `/d/` `/r/` folders were for -- makes sense. Updated!